angular-workspace/projects/ni/nimble-angular/pipes/tests/duration.pipe.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { DurationPipe } from '../duration.pipe';
+
+describe('DurationPipe', () => {
+    describe('basic formatting (English)', () => {
+        const pipe = new DurationPipe('en');
+
+        const testCases = [
+            {
+                name: 'null',
+                value: null,
+                expected: ''
+            },
+            {
+                name: 'undefined',
+                value: undefined,
+                expected: ''
+            },
+            {
+                name: '1d 2h 3m 4.56s (number)',
+                value: (4.56 + 3 * 60 + 2 * 3600 + 1 * 24 * 3600) * 1000,
+                expected: '1 day, 2 hr, 3 min, 4.56 sec'
+            },
+            {
+                name: '1d 2h 3m 4.56s (as string "93784560")',
+                value: '93784560',
+                expected: '1 day, 2 hr, 3 min, 4.56 sec'
+            },
+        ];
+
+        testCases.forEach(test => {
+            it(`${test.name} transforms to ${test.expected}`, () => {
```

Comment:
Can you use the [parameterize pattern](https://github.com/ni/nimble/blob/6db2e6b2d558cdda6457ba8744ab52f8055dd8eb/packages/nimble-components/src/utilities/tests/parameterized.ts#L73)? Makes it easier to focus/skip out a test.

Reply:
Just saw the same comment in a different thread: https://github.com/ni/nimble/pull/1619#discussion_r1369357665